.packit.yaml

```diff
@@ -0,0 +1,29 @@
+specfile_path: ansible-lint.spec
+actions:
+  create-archive:
+    - "python3 setup.py sdist --dist-dir ."
+    - "sh -c 'echo ansible-lint-$(python3 setup.py --version).tar.gz'"
+  get-current-version:
+    - "python3 setup.py --version"
+  post-upstream-clone:
+    - |
+      bash -c "sed -i -e 's/Version: VERSION_PLACEHOLDER/Version: $(python3 setup.py --version)/' ansible-lint.spec"
+jobs:
+  - job: copr_build
+    metadata:
+      targets:
+        # Unable to build centos-stream due to missing python3-setuptools_scm_git_archive
```

Comment:
Why not patch it out in the spec file? It's not like there's nowhere to get the version from.